tests/docs/source/conf.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+
+project = "autodoc_traits tests"
+extensions = ["autodoc_traits"]
```

Comment:
adding napoleon should fix issues with reading formatted docstrings, since those packages use numpy or google-format docstrings.

```suggestion
extensions = ["autodoc_traits", "sphinx.ext.napoleon"]
```